<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方案设置</string>
  </property>
  <widget class="QLineEdit" name="lenameConfig">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>方案名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>每             秒检测1次</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbfrequencyConfig">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnaddPoint">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnsave">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnexit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>391</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;点击添加按键数秒后，自动添加鼠标所在位置的坐标和颜色&lt;/p&gt;&lt;p&gt;如果是浏览器请先全屏或在使用时固定浏览器窗口大小&lt;/p&gt;&lt;p&gt;消息和消息背景颜色可直接点击表格修改(功能尚未实现)&lt;/p&gt;&lt;p&gt;自动获得的坐标有时候会错。请仔细检查&lt;/p&gt;&lt;p&gt;自动添加坐标时，表格中显示的是client坐标，弹出窗口中是screen坐标&lt;/p&gt;&lt;p&gt;被检测的窗口可以被遮挡，不能最小化。最小化的窗口不能截图&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="twpoint">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>431</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="btnDeleteRow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除所选行</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbWindowList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHwnd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>未绑定窗口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbShow">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
